services/src/main/java/org/exoplatform/task/service/LabelService.java

```diff
@@ -9,6 +9,10 @@
 import org.exoplatform.task.exception.EntityNotFoundException;
 
 public interface LabelService {
+  String LABEL_CREATION = "exo.task.labelCreation";
+
+  String LABEL_DELETED = "exo.task.labelDeleted";
```

Comment:
The name of those two listeners is not significant.
In fact, 
* `exo.task.labelCreation` will be triggered when a new labe is created
* `exo.task.labelDeleted` will be triggered when a label is deleted from Task association

Am I wrong ? If not, please consider renaming the constants and event names as well. In addition, we will have to add listeners for other events: a label is completely deleted + a label is associated to a task